docs/user-guide/102-configuration.md

```diff
@@ -476,6 +476,20 @@ node05.linux-bridge-maxunavailable   Progressing
 node06.linux-bridge-maxunavailable   Pending
 ```
 
+# Component Placement
+
+In NMState, you can constrain controllers to be assigned to specific nodes using component placement. There are several options:
+
+- **NodeSelector**
+    - Optional selector. If specified, handler will run only on nodes that have matching key-value pairs as labels applied to the node.
+- **Tolerations**
+    - Optional selector. Tolerations work together with taints to ensure that handler is not scheduled onto inappropriate nodes. One or more taints are applied to a node; this marks that the node should not accept any handler that do not tolerate the taint.
```

Comment:
Optional ~~selector~~

I would not go into that level of detail, I believe having the link to k8s documentatiton below is already enough.

`Tolerations applied to handler pods.` Should be enough I think